adafruit_bitmap_font/pcf.py

```diff
@@ -1,60 +1,160 @@
-                character = False
-            elif line.startswith("BITMAP"):
+            elif line.startswith(b"ENDCHAR"):
                 character = False
-                bitmap_lines_left = bounds[1]
-                bitmap = []
-            elif line.startswith("ENCODING"):
+                if desired_character:
+                    self._glyphs[code_point] = current_info
+                    if total_remaining == 0:
+                        return
+                desired_character = False
+            elif line.startswith(b"BBX"):
+                if desired_character:
+                    _, x, y, dx, dy = line.split()
+                    x = int(x)
+                    y = int(y)
+                    dx = int(dx)
+                    dy = int(dy)
+                    current_info["bounds"] = (x, y, dx, dy)
+                    current_info["bitmap"] = displayio.Bitmap(x, y, 2)
+            elif line.startswith(b"BITMAP"):
+                if desired_character:
+                    rounded_x = x // 8
+                    if x % 8 > 0:
+                        rounded_x += 1
+                    bytes_per_row = rounded_x
+                    if bytes_per_row % 4 > 0:
+                        bytes_per_row += 4 - bytes_per_row % 4
+                    current_y = 0
+            elif line.startswith(b"ENCODING"):
                 _, code_point = line.split()
                 code_point = int(code_point)
-                print(hex(code_point))
-            elif bitmap_lines_left > 0:
-                bits = int(line.strip(), 16)
-                shift = 8 - bounds[0]
-                bits >>= shift
-                pixels = ("{0:0" + str(bounds[0]) +"b}").format(bits).replace("0", " ")
-                bitmap.append(pixels)
-                bitmap_lines_left -= 1
-
-                if bitmap_lines_left == 0:
-                    self.characters[code_point] = {"name": character_name, "bitmap": bitmap}
+                if code_point == code_points or code_point in code_points:
+                    total_remaining -= 1
+                    if code_point not in self._glyphs:
+                        desired_character = True
+                        current_info = {"bitmap": None, "bounds": None, "shift": None}
+            elif line.startswith(b"DWIDTH"):
+                if desired_character:
+                    _, shift_x, shift_y = line.split()
+                    shift_x = int(shift_x)
+                    shift_y = int(shift_y)
+                    current_info["shift"] = (shift_x, shift_y)
+            elif line.startswith(b"SWIDTH"):
+                pass
+            elif character:
+                if desired_character:
+                    bits = int(line.strip(), 16)
+                    for i in range(rounded_x):
+                        val = (bits >> ((rounded_x-i-1)*8)) & 0xFF
+                        scratch_row[i] = val
+                    current_info["bitmap"]._load_row(current_y, scratch_row[:bytes_per_row])
```

Comment:
this looks like an added `_load_row`, which was deprecated above?

Reply:
This code doesn't work so I'm ok with it being there. I wanted to put it in the repo just in case someone wanted to pick it up and finish it.